src/Core/Addon/Theme/ThemeExporter.php

```diff
@@ -65,9 +68,11 @@ public function __construct(
     }
 
     /**
+     * Exports a theme into a zip file
+     *
      * @param Theme $theme
      *
-     * @return false|string
+     * @return string|false Path to the exported zip file
```

Comment:
```suggestion
     * @return string|bool Path to the exported zip file
```

Reply:
@sowbiba it seems that it hasn't been fixed 🤔

Reply:
Ping @sowbiba

Reply:
realpath returns string|false

https://www.php.net/manual/fr/function.realpath.php